skiko/gradle.properties

```diff
@@ -20,3 +20,4 @@ dependencies.skia.android-arm64=m110-ad42464-1
 org.gradle.jvmargs=-Xmx3G -XX:MaxMetaspaceSize=512m
 kotlin.native.cacheKind.macosX64=none
 kotlin.native.cacheKind.iosX64=none
+kotlin.mpp.enableCInteropCommonization=true
```

Comment:
It helps to build a mavenDependency

Reply:
Can we add comment why exactly it's required right in this file?

Reply:
Sure